www-client/links/links-2.19.recipe

```diff
@@ -0,0 +1,87 @@
+SUMMARY="A graphics and text mode web browser"
+DESCRIPTION="Links is a multi-platform web browser you can run in Terminal."
+HOMEPAGE="http://links.twibright.com/"
+COPYRIGHT="1999-2013 Mikulas Patocka
+	2000-2011 Petr Kulhavy, Karel Kulhavy, Martin Pergel"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://links.twibright.com/download/links-$portVersion.tar.gz"
+CHECKSUM_SHA256="37299f804f22c945bf63e22a7bc4418bbb8144b410c0ced70b82ebe6f9e3c82b"
+PATCHES="links-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 ?x86 ?x86_64"
+SECONDARY_ARCHITECTURES="?x86"
+
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	links$secondaryArchSuffix = $portVersion
+	cmd:links$commandSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libbrotlicommon$secondaryArchSuffix
+	lib:libbz2$secondaryArchSuffix
+	lib:libcrypto$secondaryArchSuffix
+	lib:libevent_2.1$secondaryArchSuffix
+	lib:libfontconfig$secondaryArchSuffix
+	lib:libfreetype$secondaryArchSuffix
+	lib:libgomp$secondaryArchSuffix
+	lib:libjpeg$secondaryArchSuffix
+	lib:libturbojpeg$secondaryArchSuffix
```

Comment:
```suggestion
```

This was not required for my local builds.

From https://github.com/haikuports/haikuports/pull/3743#issuecomment-483680331, I believe you've got a copy of libjpeg-turbo lying around in your `packages/` or have it installed, which could explain why it was used for your builds.

Reply:
Could very well be, but in the screenshot you can see it's being used?

Reply:
Maybe there should be a CONFLICTS for the provides for jpeg-turbo also (in it's recipe)?

Reply:
`links` can only use it if it's `devel` package is available during compile-time. And `jpeg-turbo` happens to also provides `devel:libjpeg` so... :p

I believe there's already a `CONFLICTS` in jpeg-turbo recipe (only for the devel package, since they don't overlap for the normal one).